pkgs/development/python-modules/nix-prefetch-github/default.nix

```
@@ -30,6 +31,8 @@ buildPythonPackage rec {
     hash = "sha256-eQd/MNlnuzXzgFzvwUMchvHoIvkIrbpGKV7iknO14Cc=";
   };
 
+  propagatedBuildInputs = [ nix ];
```

Shouldn't it be 

```suggestion
  dependencies = [ nix ];
```

?

---

changed

---

This seems wrong, it's not a python dep.

---

IIRC, I think it works, to be tested to be sure.